CONTRIBUTING.md

```diff
@@ -0,0 +1,102 @@
+# Contributing to PollCord
+
+Thank you so much for taking an interest in contributing!
+
+Follow the set of guidelines below to contribute to PollCord!
+
+## Code of Conduct
+
+By participating, you are expected to uphold this code. Please report unacceptable behavior to guptabhishek785@gmail.com, ntgomes@gmail.com
+
+Prerequisites required before starting this project during the Fall of 2022.
+
+1. Must be a graduate student at NC State University
+2. Must be a student in Software Engineering Course in Fall 2022
+3. Have proficiency in Python & Node.js
+
+
+## How can I Contribute -
+
+### Pull Requests
+
+The process described here has several goals:
+
+- Maintain the projects quality
+
+- Fix problems that are important to users
+
+- Enable a sustainable system for the projects maintainers to review contributions
+
+Please follow these steps to have your contribution reviewed by the maintainers:
+
+1. Include a clear and descriptive title.
+2. Include a description of the change.
+
+While the prerequisites above must be satisfied prior to having your pull request reviewed, the reviewer(s) may ask you to complete additional design work, tests, or other changes before your pull request can be ultimately accepted.
+
+### Adding Commands
+ Commands can be added in the form of Cogs. View hello.py as a simple example of how a Cog can be added.
+
+The basic structure is as follows:
+
+```
+class <NAME>(commands.Cog):
+
+    def __init__(self, bot):
+        self.bot = bot
+        
+    <COMMANDS RELATED TO THIS FILE>
+    
+def setup(bot):
+    bot.add_cog(<NAME>(bot))
+```
+For more information on how to use cogs, refer to the [Cogs Page](https://discordpy.readthedocs.io/en/stable/ext/commands/cogs.html)
+
+For more information on the API of discord.py you can use the [API Reference Page](https://discordpy.readthedocs.io/en/stable/api.html)
+
+For general Knowledge of discord.py use the [Documentation Page](https://discordpy.readthedocs.io/en/latest/index.html)
```

Comment:
Here you may also create another section for adding a backend. Adding commands is only for frontend... there should be something to do for backend too.